model/terms_of_service_test.go

```diff
@@ -13,19 +13,19 @@ import (
 func TestTermsOfServiceIsValid(t *testing.T) {
 	s := TermsOfService{}
 
-	assert.Error(t, s.IsValid(), "should be invalid")
+	assert.NotNil(t, s.IsValid(), "should be invalid")
 
 	s.Id = NewId()
-	assert.Error(t, s.IsValid(), "should be invalid")
+	assert.NotNil(t, s.IsValid(), "should be invalid")
 
 	s.CreateAt = GetMillis()
-	assert.Error(t, s.IsValid(), "should be invalid")
+	assert.NotNil(t, s.IsValid(), "should be invalid")
 
 	s.UserId = NewId()
-	assert.Error(t, s.IsValid(), "should be invalid")
+	assert.Nil(t, s.IsValid(), "should be valid")
```

Comment:
This was false-passing before, as `Error`, but now that is checking for `Nil` is having the expected behavior.